Services/SelectedStopPointManager.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace CanalTP\MttBundle\Services;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\Serializer\Serializer;
```

Comment:
Ok, maybe the dependency should be placed in Mtt's composer.json too ? I didn't use it because I didn't want to add a new bundle.